src/components/data-processing/CalculationConfigContainer.jsx

```diff
@@ -0,0 +1,111 @@
+import React, { useState } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import PropTypes from 'prop-types';
+import {
+  Space,
+  Form,
+  Button,
+  Alert,
+  Radio,
+} from 'antd';
+
+import _ from 'lodash';
+
+import { updateProcessingSettings } from '../../redux/actions/experimentSettings';
+
+const CalculationConfigContainer = (props) => {
+  const {
+    filterUuid, experimentId, sampleId, plotType, sampleIds, onConfigChange, children,
+  } = props;
+
+  const { auto, filterSettings: config } = useSelector(
+    (state) => state.experimentSettings.processing[filterUuid][sampleId]
+      || state.experimentSettings.processing[filterUuid],
+  );
+
+  const dispatch = useDispatch();
+
+  const [displayIndividualChangesWarning, setDisplayIndividualChangesWarning] = useState(false);
+
+  const updateAllSettings = () => {
+    setDisplayIndividualChangesWarning(false);
+
+    const newConfig = {};
+    sampleIds.forEach((currentSampleId) => {
+      newConfig[currentSampleId] = { filterSettings: config, auto };
+    });
+
+    dispatch(updateProcessingSettings(
+      experimentId,
+      filterUuid,
+      newConfig,
+    ));
+
+    onConfigChange();
+  };
+
+  const updateSettings = (diff) => {
+    const newConfig = _.cloneDeep(config);
+    _.merge(newConfig, diff);
+
+    updateSample({ [sampleId]: { auto, filterSettings: newConfig } });
+  };
+  const updateAuto = (autoMode) => {
+    updateSample({ [sampleId]: { auto: autoMode, filterSettings: config } });
+  };
+  const updateSample = (newSampleSettings) => {
+    setDisplayIndividualChangesWarning(true);
+    dispatch(updateProcessingSettings(
+      experimentId,
+      filterUuid,
+      newSampleSettings,
+    ));
+
+    onConfigChange();
+  };
+
+  return (
+    <div>
+      <Space direction='vertical' style={{ width: '100%' }} />
+      {displayIndividualChangesWarning && (
+        <Form.Item>
+          <Alert
+            message='To copy these new settings to the rest of your samples, click Copy to all samples.'
+            type='info'
+            showIcon
+          />
+        </Form.Item>
+      )}
```

Comment:
when I click from "Automatic" to "Manual", the alert doesn't disappear

Reply:
We are not tracking any of the initial state. Is it ok if I create a ticket and lave this for later?

Reply:
yes, happy to do that too

Reply:
https://biomage.atlassian.net/browse/BIOMAGE-740